Bukkit/src/main/java/com/plotsquared/bukkit/BukkitMain.java

```diff
@@ -790,7 +791,6 @@ private void startUuidCatching(@NotNull final SQLiteUUIDService sqLiteUUIDServic
                         case MUSHROOM_COW:
                         case OCELOT:
                         case PIG:
```

Comment:
This needs to be solved differently, possibly by using the names. We need this version to be backward compatible to 1.13. This should also include the new mobs, such as zombified piglins, piglins and hoglins.

Reply:
Resolved in the new commit.